tests/suites/test_suite_psa_crypto_persistent_key.data

```diff
@@ -35,7 +35,7 @@ parse_storage_data_check:"505341004b45590000000000010000000170000001000000000000
 
 # Not specific to files, but only run this test in an environment where the maximum size could be reached.
 Save maximum-size persistent raw key
```

Comment:
When `MBEDTLS_PSA_STATIC_KEY_SLOTS` is enabled, we should have a similar test case with the maximum in-memory key size plus the storage overhead.